### DISCUSSION TITLE
Penggunaan Label dalam mutate tidak keluar

### DISCUSSION THREAD

**Discussion by wildaan24** [teamalgoritma/community#926] [Q&A] [answered]
Dear Algoritma Team,

ketika masuk ke proses interactive plooting prosess tooltip dengan mengeluarkan text=label, kenapa hasilnya tidak keluar ya ?
Mohon bantuannya 
Thanks 

![image](https://github.com/user-attachments/assets/12def55b-b857-420c-b337-0abdb14b88d8)

![image](https://github.com/user-attachments/assets/720e3c7e-c280-467e-a00a-54608c5b8a90)

**Comment by IrfanChairurrachman** [comment, marked as answer]
Dear @wildaan24 ,

Bisa dipastikan di dalam fungsi `aes()` di dalam `ggplot()` telah diisi dengan `text = label`. Kemudian di `ggplotly()` diisi dengan `ggplotly(plot, tooltip = "text")`. Bukan `x` ataupun nama lainnya.

Semoga informasi ini dapat membantu. Bila ternyata informasi ini dapat menjawab pertanyaan Anda, Anda dapat “Mark as answer” jawaban ini.

**Comment by hwulanayu** [comment]
Dear @wildaan24, 

Untuk menjaga agar forum diskusi berjalan kondusif, karena tidak ada respon lanjutan setelah 7 hari dari respon terakhir, kami izin menandai pertanyaan sudah terjawab dan menutup diskusi ini. Semoga dapat dipahami dan terima kasih.

Best regards,
Team Algoritma
